wrapanapi/systems/redfish.py

```diff
@@ -13,18 +15,34 @@ class RedfishSystem(System):
         hostname: The hostname of the system.
         username: The username to connect with.
         password: The password to connect with.
+        security_protocol: The security protocol to be used for connecting with
+            the API. Expected values: 'Non-SSL', 'SSL', 'SSL without validation'
     """
 
+    # statistics for the provider
     _stats_available = {
-        'num_server': lambda self: 1,
+        'num_server': lambda self: self.num_servers(),
+    }
+
+    # statistics for an individual server
+    _server_stats_available = {
+        'cores_capacity': lambda self, requester: self.get_server_cores(requester.name),
+        'memory_capacity': lambda self, requester: self.get_server_memory(requester.name),
+        'num_firmwares': lambda self, requester: self.get_num_server_firmwares(requester.name),
```

Comment:
I had to add them because [the test implementation expects them](https://github.com/ManageIQ/integration_tests/blob/58e8401f96ec3f5fe23ad1b6ac6b508c3e6f98ae/cfme/physical/physical_server.py#L62-L63).